src/shared/constants.h

```diff
@@ -108,3 +108,6 @@ const unsigned PRIMITIVE_PORT = 42070;
 
 // the port the AI receives msgs from the robot
 const unsigned ROBOT_STATUS_PORT = 42071;
+
+// maximum transfer unit size in bytes
+const unsigned MAXIMUM_TRANSFER_UNIT_BYTES = 1500;
```

Comment:
I don't see this used anywhere? Also can the variable name specify _what_ type of transfer this refers to?

Reply:
removed it, its the MTU on the network, but ur right its not being used anywhere